sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -125,7 +129,7 @@ case class DescribeFunctionCommand(
 
           if (isExtended) {
             result :+
-              Row(s"Extended Usage:\n${replaceFunctionName(info.getExtended, info.getName)}")
+              Row(s"Extended Usage:${replaceFunctionName(info.getExtended, info.getName)}")
```

Comment:
I commented below before I saw your comment here. It's also a good place to call stripMargin :)

I don't think I understand what you mean by

> I thought it's not great when other guys fix the documentation in each function

if someone doesn't use pipes to align things, at worst `stripMargin` will have a neutral effect